tests/components/bond/test_utils.py

```diff
@@ -1,24 +0,0 @@
-"""Tests for the Bond cover device."""
-import logging
-
-from bond import Bond
-
-from homeassistant.components.bond.utils import get_bond_devices
-from homeassistant.core import HomeAssistant
-
-from tests.async_mock import patch
-
-_LOGGER = logging.getLogger(__name__)
-
-
-async def test_get_bond_devices(hass: HomeAssistant):
```

Comment:
Why are you deleting this test?

Reply:
since we no longer mock the util itself in tests (now the bond module is mocked) the util gets tested through its usage in other parts of code that are tested.

Reply:
ok